docs/dynamic-ep.rst

```diff
@@ -53,3 +53,22 @@ Manual sync of endpoints from APIC Endpoint Group to BIG-IP pool
 2. Go to L4-L7 App Services → Application Inventory → Action column and click the **Sync** icon of the desired AS3 application.
 
 3. All the operational endpoints from the mapped Tenant|App|EPG will be synced to the BIG-IP pool of the AS3 application.
+
+------
+
+FAQ
+====
+
+**Q. When new dynamic endpoints get added on APIC, the nodes aren’t getting updated on BIG-IP devices.**
```

Comment:
The Q implies this is a question, so probably should phrase this as one, for example:"
"When new dynamic endpoints are added on APIC, why aren't the nodes being updated on BIG-IP devices?"